amethyst_network/src/metrics/mod.rs

```diff
@@ -0,0 +1,59 @@
+//! Module for managing network metrics.
```

Comment:
I would like to start providing good module docs. I think even before we write a bunch of code there should be a good paragraph here explaining and an example. Not needed right now but I would like to start doing that.

Reply:
Sure, I did this since I was not sure about somethings yet.